dev-tex/biblatex: add biber use flag

lorem-ipsum opened the pull request:
biber is the default backend for biblatex for several versions now and supersedes the old bibtex and bibtex8 backends. New revision as biber is not keyworded for arm and x86 yet (a bug for that will follow once this is merged).

At the same time this patch updates to EAPI 6

mgorny commented on dev-tex/biblatex/biblatex-3.7-r1.ebuild:
```diff
@@ -0,0 +1,36 @@
+	doins -r bibtex tex
+
+	dodoc doc/latex/biblatex/{README,CHANGES.org}
+	use doc && { pushd doc/ ; latex-package_src_doinstall doc ; popd ; }
```
Nitpick: pushd, popd require ||die.

mgorny commented on dev-tex/biblatex/biblatex-3.7-r1.ebuild:
```diff
@@ -0,0 +1,36 @@
+	use doc && { pushd doc/ ; latex-package_src_doinstall doc ; popd ; }
+	if use examples ; then
+		docinto examples
+		dodoc -r doc/latex/biblatex/examples
```
Nitpick²: this creates ${docdir}/examples/examples ;-).

lorem-ipsum commented:
I have removed the examples/examples and changed the doc installation. I am not quite sure whether a separate doc use flag is really necessary, but the .tex and .pdf are quite large, increasing the install size from 1.8MB to ~4MB.
Regenerating the pdf from source is currently not possible as it requires lualatex to build and the latex-package.eclass only supports pdflatex

aballier commented on dev-tex/biblatex/biblatex-3.7-r1.ebuild:
```diff
@@ -0,0 +1,35 @@
+	doins -r bibtex tex
+
+	dodoc doc/latex/biblatex/{README,CHANGES.org}
+	use doc && dodoc doc/latex/biblatex/biblatex.{pdf,tex}
```
I think only the pdf should be installed; nobody will prefer to read the .tex :)

lorem-ipsum replied:
Yeah, you're right. I removed the .tex

aballier commented:
> Regenerating the pdf from source is currently not possible as it requires lualatex to build and the latex-package.eclass only supports pdflatex

this is not needed for merging this pr but improvements to latex-package.eclass are more than welcome!
this eclass hasn't seen new features in something like 10 years so won't support anything newer than that

lorem-ipsum commented:
I will see whether I can extend the latex-package to support lualatex and xelatex. Apparently they use different temp-paths, at least I get some sandbox violations on a quick test.

aballier commented:
just wondering: does 'texdoc biblatex' work after dropping the latex-package calls ?

lorem-ipsum commented:
of course it does not... Sorry didn't realise that the eclass is generating symlinks for texdoc. Now it is back to the original, but a bit more verbose to add the "|| die" on pushd and popd

aballier commented:
merged, thanks

please file a bug for rekeywording on arches where you dropped keywords
